catalog/spatial/registry/registry-api-impl/src/test/java/org/codice/ddf/registry/api/impl/TestRegistryStore.java

```diff
@@ -143,7 +143,7 @@ protected void validateOperation() {
             @Override
             protected SourceResponse query(QueryRequest queryRequest, ElementSetType elementSetName,
                     List<QName> elementNames, Csw csw) throws UnsupportedQueryException {
-                if(queryResults == null){
```

Comment:
That's a whitespace issue, not a package import ordering issue. Unrelated to change currently awaiting review/merge.